pymeasure/instruments/keithley/keithley2400.py

```diff
@@ -73,9 +73,15 @@ class Keithley2400(Instrument, KeithleyBuffer):
         values={'current':'CURR', 'voltage':'VOLT'},
         map_values=True
     )
-    source_enabled = Instrument.measurement("OUTPUT?",
-        """ Reads a boolean value that is True if the source is enabled. """,
-        cast=bool
+
+    source_enabled = Instrument.control(
```

Comment:
I generally prefer to have one way to do something, but I think this is a useful addition.